phpstan.neon.dist

```diff
@@ -0,0 +1,5 @@
+parameters:
+  level: 3
```

Comment:
Should probably be increased to 7, but I didn't want to add that to this PR. Level 3 is the level which was configured as a command line argument in the Travis CI configuration.

Reply:
Yes, let's try to be as close to Travis as possible 👍